docs/integrations/prometheus.md

```diff
@@ -92,3 +92,11 @@ remote_read:
 remote_write:
   - url: "http://localhost:7201/api/v1/prom/remote/write"
 ```
+
+Also, if you want to see metrics generated by the `m3coordinator` or `m3query` process such as number of queries, etc., you should add the following job under `scrape_configs`:
```

Comment:
How about:

"Also, we recommend adding M3DB and `M3Coordinator`/`M3Query' to your list of jobs so that you can monitor them with Prometheus. With Prometheus scraping setup,  you can also use our pre-configured M3DB [Grafana dashboard](https://grafana.com/dashboards/8126)"